lib/docker.js

```diff
@@ -206,7 +206,7 @@ App.prototype.setupConnections = function setupConnections() {
         }
     });
 
-    if (self.config.experimental_nfs_shared_volumes === true) {
+    if (self.config.experimental_docker_nfs_shared_volumes === true) {
```

Comment:
Once again, a beautiful change ruined by the fact that we put the copyright date in our files to make sure they need to be updated every year. Will we update all 2016 files before it's 2018? I doubt it!